app/models/services/facebook.rb

```diff
@@ -24,14 +24,18 @@ def post_to_facebook(url, body)
 
   def create_post_params(post)
     message = post.message.plain_text_without_markdown
-    if post.photos.any?
-      message += " " + short_post_url(post, protocol: AppConfig.pod_uri.scheme,
-                                            host: AppConfig.pod_uri.authority)
-    end
-
-    {message: message,
-     access_token: access_token,
-     link:  post.message.urls.first
+    post_url = short_post_url(post, protocol: AppConfig.pod_uri.scheme, host: AppConfig.pod_uri.authority)
+
+    message = if message.empty? && post.photos.any?
+                post_url
+              else
+                "#{message} (via #{post_url})"
+              end
+
+    {
+      message: message,
```

Comment:
Layout/AlignHash: Align the elements of a hash literal if they span more than one line.